python/jupytergis_qgis/jupytergis_qgis/qgis_loader.py

```diff
@@ -177,11 +182,15 @@ def qgis_layer_to_jgis(
             color["stroke-color"] = symbol.color().name() + alpha
 
         if isinstance(symbol, QgsFillSymbol):
-            color["fill-color"] = symbol.color().name() + alpha
+            color["fill-color"] = symbol.color().name()
+            color["stroke-color"] = symbol.color().name()
```

Comment:
```suggestion
            color["fill-color"] = symbol.color().name() + alpha
            color["stroke-color"] = symbol.color().name() + alpha
```

I'm assuming we want this to have the alpha values like the Marker and Line symbols?

Reply:
I removed these alpha values because it was resulting in [these hex strings](https://github.com/geojupyter/jupytergis/actions/runs/13032409740/job/36354496301?pr=390#step:6:403) which were resulting in Assertion Error